### DISCUSSION TITLE
Yazi as a file picker doesn't output to stdout

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [sxyazi/yazi#823] [1. Q&A]
`yazi --cwd-file=/dev/stdout `
Is the above command not outputting anything expected?

**Comment by mikavilpas** [comment]
Not sure if it's related but I think it was changed to output to stderr yesterday.

**Reply by LamprosPitsillos** [reply]
This is is happening on 2.4 stable

**Reply by mikavilpas** [reply]
Ok it's probably unrelated then. I built from source yesterday to test it, and what I have is this:

```
$ yazi --version
yazi 0.2.4 (411ba2f 2024-03-15)
```

**Comment by sxyazi** [comment]
Yazi added support for `stdout` yesterday, with the latest main branch, you can use

```bash
yazi --cwd-file /dev/stdout | cat
```

to get the CWD when exiting.


https://github.com/sxyazi/yazi/assets/17523360/7381010d-fcd4-4f4c-9b12-84359da61b69

**Reply by LamprosPitsillos** [reply]
Thanks!
Why didn't it work in the first place though? Since the file it should have outputted to was essentially stdout
